net-dialup/freeradius-client/freeradius-client-1.1.7-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="FreeRADIUS Client framework"
+HOMEPAGE="https://wiki.freeradius.org/project/Radiusclient"
+SRC_URI="ftp://ftp.freeradius.org/pub/freeradius/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+IUSE="scp shadow static-libs"
```

Comment:
What does `static-libs` IUSE do in this ebuild? Should it somehow work in `econf`?

Reply:
By reading https://devmanual.gentoo.org/eclass-reference/autotools-utils.eclass/index.html#lbAF it seems that is was directly handled by the old eclass 

> autotools-utils_src_install
>     The autotools src_install function. Runs emake install, unconditionally removes unnecessary static libs (based on shouldnotlink libtool property) and removes unnecessary libtool files when static-libs USE flag is defined and unset.

So I added this in src_install:
```
    use static-libs || \
        find "${ED}" -name '*.a' -delete
```

It seems to be how it’s handled by reading other ebuilds.

It still installs `/usr/lib64/libfreeradius-client.la`, but it’s not installed by the current ebuild (with jeunet being my dev box and lns01 a production one):
```
alarig@jeunet | net-dialup/freeradius-client *% freeradius-client % ls -lh /var/tmp/portage/net-dialup/freeradius-client-1.1.7-r1/image/usr/lib64/libfreeradius-client.*
-rw-r--r-- 1 alarig alarig 1017 May 20 22:00 /var/tmp/portage/net-dialup/freeradius-client-1.1.7-r1/image/usr/lib64/libfreeradius-client.la
lrwxrwxrwx 1 alarig alarig   29 May 20 22:00 /var/tmp/portage/net-dialup/freeradius-client-1.1.7-r1/image/usr/lib64/libfreeradius-client.so -> libfreeradius-client.so.2.0.0
lrwxrwxrwx 1 alarig alarig   29 May 20 22:00 /var/tmp/portage/net-dialup/freeradius-client-1.1.7-r1/image/usr/lib64/libfreeradius-client.so.2 -> libfreeradius-client.so.2.0.0
-rwxr-xr-x 1 alarig alarig  61K May 20 22:00 /var/tmp/portage/net-dialup/freeradius-client-1.1.7-r1/image/usr/lib64/libfreeradius-client.so.2.0.0

alarig@lns01 ~ $ ls -lh /usr/lib64/libfreeradius-client.*
lrwxrwxrwx 1 root root  29 Oct  8  2018 /usr/lib64/libfreeradius-client.so -> libfreeradius-client.so.2.0.0
lrwxrwxrwx 1 root root  29 Oct  8  2018 /usr/lib64/libfreeradius-client.so.2 -> libfreeradius-client.so.2.0.0
-rwxr-xr-x 1 root root 57K Oct  8  2018 /usr/lib64/libfreeradius-client.so.2.0.0
```

But as I’m not very sure if it’s needed by *.so, I kept it a the moment (there are several *.la on all my boxes).